### PR TITLE
Introduce push type field in send_message

### PR DESCRIPTION
**Pull request by nikitph**
self explanatory

### CODE REVIEW

File: kalyke/client.py [hunk 1]
```diff
@@ -121,7 +121,8 @@ async def _send_message(self, registration_id, alert, identifier=None, expiratio
         headers = {
             'apns-expiration': str(expiration_time),
             'apns-priority': str(priority),
-            'apns-topic': topic
+            'apns-topic': topic,
+            'apns-push-type': push_type
```
Comment (nnsnodnb) [inline kalyke/client.py:125]:
```suggestion
            'apns-push-type': push_type,
```

I also forgot to put a comma after the last element.

**Comment by nnsnodnb** [comment]
What about background and voip apns-push-type with this change?

**Comment by nikitph** [comment]
That's something they will have to pass in the send message function as the
param push_type. We are just defaulting to alert.

**Comment by nnsnodnb** [comment]
For `send_bulk_message`, `push_type` is pinned to `alert`.

https://github.com/nikitph/kalyke/blob/2803864164da1fb7f4ecb8295e3d5992246bcae8/kalyke/client.py#L95-L100

I want to following code.

```python
push_type = kwargs.get('push_type')
```

**Pull request closed without merging**